package.json

```diff
@@ -4,10 +4,12 @@
   "description": "A container for large mutable strings with annotated regions",
   "main": "./lib/text-buffer",
   "scripts": {
-    "prepublish": "grunt clean lint coffee shell:update-atomdoc atomdoc",
-    "compile": "coffee --no-header --output lib --compile src",
-    "lint": "coffeelint -r src spec",
-    "test": "jasmine-focused --coffee --captureExceptions --forceexit spec",
+    "prepublish": "npm run clean && npm run compile && npm run lint && npm run atomdoc",
+    "atomdoc": "grunt shell:update-atomdoc atomdoc",
```

Comment:
If there's a way to compile atomdoc outside grunt we could kill off grunt completely.